test/Data/Source/Spec.hs

```diff
@@ -1,53 +1,77 @@
-module Data.Source.Spec (spec) where
+{-# LANGUAGE NamedFieldPuns #-}
+module Data.Source.Spec (spec, testTree) where
 
-import Data.Char (chr)
-import Data.Functor.Listable
 import Data.Range
 import Data.Source
 import Data.Span
 import qualified Data.Text as Text
+
 import Test.Hspec
-import Test.Hspec.LeanCheck
-import Test.LeanCheck
+
+import qualified Generators as Gen
+import qualified Hedgehog.Gen as Gen
+import           Hedgehog ((===))
+import qualified Hedgehog.Range
+import           Hedgehog hiding (Range)
+import qualified Test.Tasty as Tasty
+import           Test.Tasty.Hedgehog (testProperty)
+
+prop :: HasCallStack => String -> (Source -> PropertyT IO ()) -> Tasty.TestTree
```

Comment:
Wrote this little helper to save keystrokes and modify as few of the specs as possible.